apps/app-get-started/src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue'
+
+declare global {
+  namespace JSX {
+    // tslint:disable no-empty-interface
+    interface Element extends VNode {}
+    // tslint:disable no-empty-interface
+    interface ElementClass extends Vue {}
+    interface IntrinsicElements {
+      [elem: string]: any
```

Comment:
Aah, is this why there is typescript? So we can get the shims? Do we actually need these?